packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -122,7 +123,8 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
-            history.push(routes.cashier_withdrawal);
+            const is_next_wallet = localStorage.getObject('FeatureFlagsStore')?.data.next_wallet;
```

Comment:
Better to put this out of this case and put it up. Incase other cases needs to use this in the future

Reply:
Updated